ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_fft.py

```diff
@@ -269,6 +293,7 @@ def test_jax_numpy_rfft(
     dtype_input_axis, n, norm, frontend, backend_fw, test_flags, fn_tree, on_device
 ):
     input_dtype, x, axis = dtype_input_axis
+
```

Comment:
Could you please remove this extra line added here as it is not related to the function are working on?

Reply:
I'll do it right away

Reply:
I have removed the extra code, please check,
Thanks for your patience 😊